lib/active_admin_import/dsl.rb

```diff
@@ -31,11 +32,12 @@ module DSL
       if result.empty?
         flash[:warning] = I18n.t('active_admin_import.file_empty_error')
       else
-        if result.has_imported?
-          flash[:notice] = I18n.t('active_admin_import.imported', count: result.imported_qty, model: model_name, plural_model: plural_model_name)
-        end
         if result.has_failed?
           flash[:error] = I18n.t('active_admin_import.failed', count: result.failed.count, model: model_name, plural_model: plural_model_name)
+          return if options[:batch_transaction]
+        end
+        if result.has_imported?
+          flash[:notice] = I18n.t('active_admin_import.imported', count: result.imported_qty, model: model_name, plural_model: plural_model_name)
```

Comment:
Line is too long. [145/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.